docs/Actions/load-user.md

```diff
@@ -0,0 +1,166 @@
+---
+id: LoadUser
+title: Load User
+sidebar_label: Load User
+---
+
+An action that allows altering the user which the subsequent actions refer to. The user change affects all actions after the `Load User` action and after all actions have finished executing the user will be reverted back to the initial one (i.e. the logged-in user). By default, all actions that need a user for their execution will refer to the currently logged in user.
+
+This action also supports loading multiple users by providing multiple user identifiers. The last user identifier provided will also be used as the context user under which all actions execute. Loading multiple users creates a hidden context entity that is leveraged by the following actions which support this:
+
+- [Grant Role](/docs/Actions/grant-role.md)
+- [Revoke Role](/docs/Actions/revoke-role.md)
+- [Authorize User](/docs/Actions/authorize-user.md)
+- [Unauthorize User](/docs/Actions/unauthorize-user.md)
+- [Delete User](/docs/Actions/delete-user.md)
+- [Send Email](/doc/Actions/send-email.md).
+
+When used in conjunction with the actions above, the logic applies to all users that have been loaded into the context by the `Load User` action. For example, the `Grant Role` action will assign the specified role to each and every user loaded into the context.
+
+Calling the `Load User` action the second time inside the same workflow will replace the context user that the actions use and add to the list of context users (the Users entity) that actions such as `Grant Role` leverage. If a user is already present in the Users entity he will not be added again.
```

Comment:
Avoid "he" or "she" as it discriminates. In this case "it" works. Or, prefer plural, as "they" is gender-neutral.